alert.tf

```diff
@@ -16,9 +16,9 @@ resource "azurerm_monitor_activity_log_alert" "main" {
   count               = var.alert_limit_reached ? 0 : 1
   name                = "Application Insights daily cap reached - ${local.name}"
   resource_group_name = var.resource_group_name
-  location            = var.alert_location
+  location            = var.location
```

Comment:
alert_location is correct afaik - https://github.com/hmcts/terraform-module-application-insights/blob/e1b7f3ff0c0e0580657f08fc16a96aab94165b17/inputs-optional.tf#L54-L58